### ISSUE TITLE
Crashes when loading official Mission Paks

### ISSUE THREAD

**Issue by Sisee84** [Rinnegatamante/vitaQuake#77]
I know that the mission packs worked in the past, as I distinctly remember loading them up.
However, only the core game works now.

When selecting the Live Area shortcuts for either mission pak, the terminal loads fine, but then crashes, giving an error message:
"An error has occurred in the following application..."
And proceeds to save a Core file.

It appears that the program crashes upon trying to download assets.
As the last line of code in the games terminal screen reads:
"WEB DOWNLOADING: PROGS/EMFORCER.MDL FROM HTTP://BIGFOOT.SERVEQUAKE.COM/PROGS/ENFORCER.MDL"

**Comment by Rinnegatamante** [comment]
Do you have mission pack data files? If so, are they FULL lowercases? (extensions too)

**Comment by Sisee84** [comment]
I placed the mission pak folders as such:.
Scourge of Armagon: "ux0:data/Quake/hipnotic"
Dissolution of Eternity: "ux0:data/Quake/rogue"

The files inside each folder are labeled as "pak0.pak", in lowercase as they are in the PC version I have installed.
I can try relabeling them in uppercase, however the pak files do seem to load up, the program only crashes when trying to retrieve some files online. As shown in the screenshot I've attached.
